.gitignore

```diff
@@ -0,0 +1,27 @@
+/logs
+/target
+/project/project
+/project/target
+/.idea
+/.idea_modules
+/.classpath
+/.project
+/.settings
+/RUNNING_PID
+*~
+bin/
+.cache-main
+.cache-tests
+/db
+.eclipse
+/lib/
+/logs/
+/modules
+tmp/
+test-result
+server.pid
+*.iml
+*.eml
+/dist/
+.cache
+evolutions/
```

Comment:
Ok, I didn't notice the context of the comment. I think that we should be committing evolutions.